python/cuml/cluster/dbscan.pyx

```diff
@@ -204,19 +205,17 @@ class DBSCAN(Base):
         if self.max_mbytes_per_batch is None:
             self.max_mbytes_per_batch = 0
 
+    @generate_docstring(skip_parameters_heading=True)
```

Comment:
I want to avoid the parsing to reduce the time as I mentioned above, looking out of order might be a minor sacrifice to keep this as fast as possible. One of the trickier parts has been balancing capabilities of the decorator with not impacting import time of the module